src/uct/api/v2/uct_v2.h

```diff
@@ -671,7 +671,10 @@ typedef enum uct_md_attr_field {
     UCT_MD_ATTR_FIELD_EXPORTED_MKEY_PACKED_SIZE = UCS_BIT(13),
 
     /** Unique global identifier of the memory domain. */
-    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(14)
+    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(14),
+
+    /** Indicate memory types that are ODPable */
+    UCT_MD_ATTR_FIELD_ODP_REG_MEM_TYPES         = UCS_BIT(15)
```

Comment:
let's rename it according to the API, e.g. `UCT_MD_ATTR_FIELD_NONBLOCK_REG_MEM_TYPES` or `UCT_MD_ATTR_FIELD_NB_REG_MEM_TYPES`
and the field itself as well